MAUI/ImageEditor/getting-started.md

```diff
@@ -252,5 +252,56 @@ The [`GetImageStream`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Image
         this.imageEditor.GetImageStream();
     }
 
+{% endhighlight %}
+{% endtabs %}
+
+## Get the Image Original Size
+The [`OriginalImageSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_OriginalImageSize) property is used to get the image's original size.
+

```

Comment:
Note : The size value will be available only after image loaded into view

Reply:
note added